src/settings.rs

```diff
@@ -65,10 +66,46 @@ example:
         assert!(!policies.is_empty());
 
         let policy = policies.get("example").unwrap();
+        assert_eq!(policy.allowed_to_mutate, Option::default());
```

Comment:
nitpicking, maybe inside of the test it would make more sense to be explicit about what we expect the value to be:
replace  `Option::default()` with `None`.